### PR TITLE
[4.0] Namespace libraries/src

### PR DESCRIPTION
**Pull request by brianteeman**
Continuing the namespace work

NOTE there are a lot of files in this folder and its hard to separate into smaller pr so I have decided to do it in batches ie test and merge this one which is the main namespacing and then do subsequent pr once merged for the remaining namespaces

### CODE REVIEW

File: libraries/src/Application/CMSApplication.php [hunk 1]
```diff
@@ -596,7 +597,7 @@ public function getUserState($key, $default = null)
 	 * @param   string  $key      The key of the user state variable.
 	 * @param   string  $request  The name of the variable passed in a request.
 	 * @param   string  $default  The default value for the variable if not found. Optional.
-	 * @param   string  $type     Filter for the variable, for valid values see {@link \JFilterInput::clean()}. Optional.
+	 * @param   string  $type     Filter for the variable, for valid values see {@link InputFilter::clean()}. Optional.
```
Comment (wilsonge) [inline libraries/src/Application/CMSApplication.php:601]: @mbabker does this need to be the fully qualified class for the `@link` tag to work or will just a normal `use` statement work?

Comment (brianteeman) [reply libraries/src/Application/CMSApplication.php:601]: yeah - i have no idea what i am doing - just trying to do the grunt work

Comment (mbabker) [reply libraries/src/Application/CMSApplication.php:601]: It's fine, see https://api.joomla.org/cms-3/classes/Joomla.CMS.Factory.html#method_getApplication

**Comment by brianteeman** [comment]
(currently unable to connect to ci.joomla.org to see what the drone errors are)

**Comment by rdeutz** [comment]
code style issues are the reason

**Comment by brianteeman** [comment]
I will have to look at it another day as for some reason my internet connection will not route to ci.joomla.ory so that I can see the issues

**Comment by rdeutz** [comment]
I can help :-)

```

FILE: ...thub.com/joomla/joomla-cms/libraries/src/Installer/InstallerAdapter.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 125 | ERROR | Expected 2 spaces after the longest type
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------


FILE: ...ne/src/github.com/joomla/joomla-cms/libraries/src/Schema/ChangeItem.php
--------------------------------------------------------------------------------
FOUND 2 ERROR(S) AFFECTING 2 LINE(S)
--------------------------------------------------------------------------------
 120 | ERROR | The variable names for parameters $db (1) and $file (2) do not
     |       | align
 137 | ERROR | The variable names for parameters $db (1) and $file (2) do not
     |       | align
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------


FILE: ...one/src/github.com/joomla/joomla-cms/libraries/src/Schema/ChangeSet.php
--------------------------------------------------------------------------------
FOUND 2 ERROR(S) AFFECTING 2 LINE(S)
--------------------------------------------------------------------------------
  66 | ERROR | The variable names for parameters $db (1) and $folder (2) do not
     |       | align
 143 | ERROR | The variable names for parameters $db (1) and $folder (2) do not
     |       | align
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------


FILE: /drone/src/github.com/joomla/joomla-cms/libraries/src/UCM/UCMType.php
--------------------------------------------------------------------------------
FOUND 2 ERROR(S) AFFECTING 2 LINE(S)
--------------------------------------------------------------------------------
 86 | ERROR | The variable names for parameters $alias (1) and $database (2) do
    |       | not align
 87 | ERROR | The variable names for parameters $database (2) and $application
    |       | (3) do not align
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------


FILE: /drone/src/github.com/joomla/joomla-cms/libraries/src/Date/Date.php
--------------------------------------------------------------------------------
FOUND 1 ERROR(S) AFFECTING 1 LINE(S)
--------------------------------------------------------------------------------
 429 | ERROR | The variable names for parameters $local (1) and $db (2) do not
     |       | align
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------


FILE: /drone/src/github.com/joomla/joomla-cms/libraries/src/Table/Table.php
--------------------------------------------------------------------------------
FOUND 2 ERROR(S) AFFECTING 2 LINE(S)
--------------------------------------------------------------------------------
 140 | ERROR | The variable names for parameters $key (2) and $db (3) do not
     |       | align
 141 | ERROR | The variable names for parameters $db (3) and $dispatcher (4) do
     |       | not align
--------------------------------------------------------------------------------
UPGRADE TO PHP_CODESNIFFER 2.0 TO FIX ERRORS AUTOMATICALLY
--------------------------------------------------------------------------------
```

**Comment by brianteeman** [comment]
It will still have to wait until tomorrow as I am out for the evening. Unless you want to do a pr to my branch:)

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks. Now to re-review this folder for the less common stuff

**Pull request merged**